packages/vx-demo/src/pages/chord.js

```diff
@@ -19,23 +19,15 @@ import { Arc } from '@vx/shape';
 import { Group } from '@vx/group';
 import { Chord, Ribbon } from '@vx/chord';
 import { scaleOrdinal } from '@vx/scale';
-import { LinearGradient } from '@vx/gradient';
+import { schemeDark2 } from 'd3-scale-chromatic';
```

Comment:
could we remove the changes in the page source code as well? 

sorry this is more work than it should be, we're scheming ways to improve docs + examples in the near future after TS wraps up.

Reply:
Sorry, forgot about that one. Updated the page source code.